index.js

```diff
@@ -1,13 +1,106 @@
 const express = require('express');
 const helmet = require('helmet');
 
+
 const server = express();
+const knex = require('knex')
 
 server.use(express.json());
 server.use(helmet());
 
+const knexConfig = {
+  client: 'sqlite3',
+  useNullAsDefault: true,
+  connection: {
+    filename: './data/lambda.sqlite3'
+  }
+}
+
+const db = knex(knexConfig)
 // endpoints here
 
+server.get('/api/zoos/', (req, res)  => {
+  db('zoos')
+  .then(zoos => {
+    res.status(200).json(zoos)
+  })
+  .catch(error => {
+    res.status(500).json(error)
+  })
+});
+
+
+server.get('/api/zoos/:id', (req, res)  => {
```

Comment:
Even though we're handling data access inside the function, we would still like to do conditionals to check if we actually got the id that was passed in the params when we did our where check.